editor/js/Menubar.Help.js

```diff
@@ -54,6 +54,18 @@ function MenubarHelp( editor ) {
 	} );
 	options.add( option );
 
+	// Manual
+
+	var option = new UIRow();
```

Comment:
I personally think we should avoid var as much as possible.

Reply:
You are right but I'm trying to be consistent since most editor code (including `Menubar.Help.js`) does not yet use `let`/`const`.